.github/workflows/backend_dev.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: python-app
+          include-hidden-files: true
```

Comment:
## The fix
This is a new param I have to add as of v4.

The original cause:
- https://github.com/jhu-bids/TermHub/pull/889
- Migration from v2 to v4 https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/